classrooms/views.py

```
@@ -2,75 +2,51 @@
 from django.contrib.auth.decorators import login_required
 from django.contrib import messages
 from django.urls import reverse
+from django.views.decorators.http import require_http_methods
+from django.http import HttpRequest, HttpResponse, HttpResponseNotAllowed
```

Ruff (0.8.2)

`django.http.HttpResponseNotAllowed` imported but unused

Remove unused import: `django.http.HttpResponseNotAllowed`

(F401)